.changelog/README.md

```diff
@@ -0,0 +1,132 @@
+# How To Use
+
+Nomad uses [`go-changelog`](https://github.com/hashicorp/go-changelog) to generate its changelog on release.
+
+To install, run the following command:
+
+```
+go get github.com/hashicorp/go-changelog/cmd/changelog-build
+```
+
+## Developer Guide
+
+PRs with code changes should include a `.changelog/<PR Number>.txt` file highlighting the user visible change.
+
+The changelog entry should include the affected component (e.g. `driver/docker`, `api`), and a user-centric summary of the change. If an issue warrants more elaborate description, also update the [Upgrade Guide](../website/content/docs/upgrade/upgrade-specific.mdx).
+
+Use the PR number as the file name. Enterprise private changes should have an entry in the OSS repository referencing a public GitHub issues detailing the change and impact.
+
+Below are some examples of how to generate a CHANGELOG entry with your pull
+request.
+
+### Improvement
+
+~~~
+```release-note:improvement
```

Comment:
The syntax here is confusing to me... are the `.txt` files supposed to have Markdown syntax? If so, we should say that explicitly.

Reply:
`go-changelog` just happened to use this markdown-like syntax but the tool doesn't expect or require markdown: https://github.com/hashicorp/go-changelog/blob/89e3f9b779494b52e013b9615f1a7d8937fca0a8/note.go#L15-L20 .

Reply:
Wild. 😁 Maybe we can call out what we'd like folks to do for consistency anyways?